nqp/core/definitions.py

```diff
@@ -1,9 +1,7 @@
 from __future__ import annotations
 
-import logging
-from typing import List, Tuple, TYPE_CHECKING, Union
+from typing import Tuple
 
-from pygame import Vector2
+__all__ = ["TileLocation"]
```

Comment:
this type is required because a pygame Vector2 cannot be used as dictionary keys